<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_bill_Dialog</class>
 <widget class="QDialog" name="edit_bill_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>585</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>10</y>
     <width>20</width>
     <height>561</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>11</y>
     <width>252</width>
     <height>564</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Patient's Name:</string>
      </property>
      <widget class="QComboBox" name="comboBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Bill No:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>RATE:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>QTY:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Discount:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Extra:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="title">
       <string>Date:</string>
      </property>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>230</width>
         <height>31</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>261</width>
     <height>561</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>Total</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>230</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 28pt &quot;MS Shell Dlg 2&quot;;</string>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>70</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 36pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="title">
       <string>ADVANCE:</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>61</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>230</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Finish</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
